Welcome to modernity with GVfs and KDE4

--- origsrc/xdg-utils-1.0.2/scripts/xdg-email	2007-06-24 14:58:04.000000000 -0500
+++ src/xdg-utils-1.0.2/scripts/xdg-email	2009-11-12 02:31:51.708615000 -0600
@@ -392,10 +392,9 @@ open_kde()
             fi
         fi
     fi
-    DEBUG 1 "Running kmailservice \"$1\""
+    DEBUG 1 "Running kde-open \"$1\""
     # KDE uses locale's encoding when decoding the URI, so set it to UTF-8
-    LC_ALL=C.UTF-8 kmailservice "$1"
-    kfmclient_fix_exit_code $?
+    LC_ALL=C.UTF-8 kde-open "$1"
 
     if [ $? -eq 0 ]; then
         exit_success
@@ -406,8 +405,8 @@ open_kde()
 
 open_gnome()
 {
-    DEBUG 1 "Running gnome-open \"$1\""
-    gnome-open "$1"
+    DEBUG 1 "Running gvfs-open \"$1\""
+    gvfs-open "$1"
 
     if [ $? -eq 0 ]; then
         exit_success
--- origsrc/xdg-utils-1.0.2/scripts/xdg-open	2007-06-24 14:58:01.000000000 -0500
+++ src/xdg-utils-1.0.2/scripts/xdg-open	2009-11-12 02:24:39.258260900 -0600
@@ -326,8 +326,7 @@ kfmclient_fix_exit_code()
 
 open_kde()
 {
-    kfmclient exec "$1"
-    kfmclient_fix_exit_code $?
+    kde-open "$1"
 
     if [ $? -eq 0 ]; then
         exit_success
@@ -338,7 +337,7 @@ open_kde()
 
 open_gnome()
 {
-    gnome-open "$1"
+    gvfs-open "$1"
 
     if [ $? -eq 0 ]; then
         exit_success
