--- origsrc/xdg-utils-1.0.2/scripts/xdg-email	2007-06-24 14:58:04.000000000 -0500
+++ src/xdg-utils-1.0.2/scripts/xdg-email	2008-07-09 00:01:48.531250000 -0500
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #---------------------------------------------
 #   xdg-email
 #
@@ -435,7 +435,8 @@
     for browser in $BROWSER; do
         if [ x"$browser" != x"" ]; then
 
-            browser_with_arg=`echo "$browser" | sed s#%s#"$1"#`
+            IFS=' '
+            browser_with_arg=${browser//'%s'/"$1"}
 
             if [ x"$browser_with_arg" = x"$browser" ]; then "$browser" "$1";
             else $browser_with_arg;
--- origsrc/xdg-utils-1.0.2/scripts/xdg-open	2007-06-24 14:58:01.000000000 -0500
+++ src/xdg-utils-1.0.2/scripts/xdg-open	2008-07-09 00:10:41.593750000 -0500
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #---------------------------------------------
 #   xdg-open
 #
@@ -364,7 +364,8 @@
     for browser in $BROWSER; do
         if [ x"$browser" != x"" ]; then
 
-            browser_with_arg=`echo "$browser" | sed s#%s#"$1"#`
+            IFS=' '
+            browser_with_arg=${browser//'%s'/"$1"}
 
             if [ x"$browser_with_arg" = x"$browser" ]; then "$browser" "$1";
             else $browser_with_arg;
