--- origsrc/xdg-utils-1.1.1/scripts/xdg-email.in	2015-10-05 13:59:19.000000000 -0500
+++ src/xdg-utils-1.1.1/scripts/xdg-email.in	2015-10-07 21:09:37.390445000 -0500
@@ -434,7 +434,7 @@ fi
 if [ x"$BROWSER" = x"" ]; then
     BROWSER=www-browser:links2:elinks:links:lynx:w3m
     if [ -n "$DISPLAY" ]; then
-        BROWSER=x-www-browser:firefox:iceweasel:seamonkey:mozilla:epiphany:konqueror:chromium-browser:google-chrome:$BROWSER
+        BROWSER=epiphany:midori:konqueror:rekonq:qupzilla:$BROWSER
     fi
 fi
 
--- origsrc/xdg-utils-1.1.1/scripts/xdg-open.in	2015-10-05 13:59:19.000000000 -0500
+++ src/xdg-utils-1.1.1/scripts/xdg-open.in	2015-10-07 21:09:54.189405900 -0500
@@ -380,7 +380,7 @@ open_generic()
     if [ x"$BROWSER" = x"" ]; then
         BROWSER=www-browser:links2:elinks:links:lynx:w3m
         if [ -n "$DISPLAY" ]; then
-            BROWSER=x-www-browser:firefox:iceweasel:seamonkey:mozilla:epiphany:konqueror:chromium-browser:google-chrome:$BROWSER
+            BROWSER=epiphany:midori:konqueror:rekonq:qupzilla:$BROWSER
         fi
     fi
 
