--- origsrc/xdg-utils-1.1.1/scripts/xdg-copy.in	2015-10-05 13:59:19.000000000 -0500
+++ src/xdg-utils-1.1.1/scripts/xdg-copy.in	2015-10-07 21:16:02.278459400 -0500
@@ -96,7 +96,7 @@ case "$DE" in
     copy_kde "$source" "$dest"
     ;;
 
-    gnome*|cinnamon|lxde|mate|xfce)
+    gnome*|cinnamon|lxde|mate|xfce|cygwinx)
     copy_gnome "$source" "$dest"
     ;;
 
--- origsrc/xdg-utils-1.1.1/scripts/xdg-email.in	2015-10-07 21:10:45.159321200 -0500
+++ src/xdg-utils-1.1.1/scripts/xdg-email.in	2015-10-07 21:16:02.282459600 -0500
@@ -451,7 +451,7 @@ case "$DE" in
     open_gnome "${mailto}"
     ;;
 
-    gnome3|cinnamon|lxde|mate)
+    gnome3|cinnamon|lxde|mate|cygwinx)
     open_gnome3 "${mailto}"
     ;;
 
--- origsrc/xdg-utils-1.1.1/scripts/xdg-file-dialog.in	2015-10-05 13:59:19.000000000 -0500
+++ src/xdg-utils-1.1.1/scripts/xdg-file-dialog.in	2015-10-07 21:16:02.288460000 -0500
@@ -312,7 +312,7 @@ if [ "$action" = "openfilename" ]; then
         open_kde "$filename"
         ;;
 
-        gnome*|cinnamon|lxde|mate|xfce)
+        gnome*|cinnamon|lxde|mate|xfce|cygwinx)
         open_zenity "$filename"
         ;;
 
@@ -326,7 +326,7 @@ elif [ "$action" = "openfilenamelist" ];
         open_multi_kde "$filename"
         ;;
 
-        gnome*|cinnamon|lxde|mate|xfce)
+        gnome*|cinnamon|lxde|mate|xfce|cygwinx)
         open_multi_zenity "$filename"
         ;;
 
@@ -340,7 +340,7 @@ elif [ "$action" = "savefilename" ]; the
         save_kde "$filename"
         ;;
 
-        gnome*|cinnamon|lxde|mate|xfce)
+        gnome*|cinnamon|lxde|mate|xfce|cygwinx)
         save_zenity "$filename"
         ;;
 
@@ -354,7 +354,7 @@ elif [ "$action" = "directory" ]; then
         directory_kde "$filename"
         ;;
 
-        gnome*|cinnamon|lxde|mate|xfce)
+        gnome*|cinnamon|lxde|mate|xfce|cygwinx)
         directory_zenity "$filename"
         ;;
 
--- origsrc/xdg-utils-1.1.1/scripts/xdg-mime.in	2015-10-05 13:59:19.000000000 -0500
+++ src/xdg-utils-1.1.1/scripts/xdg-mime.in	2015-10-07 21:16:02.292460200 -0500
@@ -586,7 +586,7 @@ if [ "$action" = "info" ]; then
         info_kde "$filename"
         ;;
 
-        gnome*|cinnamon|lxde|mate|xfce)
+        gnome*|cinnamon|lxde|mate|xfce|cygwinx)
         info_gnome "$filename"
         ;;
 
--- origsrc/xdg-utils-1.1.1/scripts/xdg-open.in	2015-10-07 21:10:45.162321400 -0500
+++ src/xdg-utils-1.1.1/scripts/xdg-open.in	2015-10-07 21:16:02.313461400 -0500
@@ -460,7 +460,7 @@ case "$DE" in
     open_kde "$url"
     ;;
 
-    gnome3|cinnamon)
+    gnome3|cinnamon|cygwinx)
     open_gnome3 "$url"
     ;;
 
--- origsrc/xdg-utils-1.1.1/scripts/xdg-utils-common.in	2015-10-05 13:59:19.000000000 -0500
+++ src/xdg-utils-1.1.1/scripts/xdg-utils-common.in	2015-10-07 21:16:59.665741700 -0500
@@ -279,6 +279,9 @@ detectDE()
          XFCE)
            DE=xfce
            ;;
+         X-Cygwin)
+           DE=cygwinx
+           ;;
          X-Generic)
            DE=generic
            ;;
