--- origsrc/xdg-utils/scripts/xdg-copy.in	2015-09-18 04:14:28.000000000 -0500
+++ src/xdg-utils/scripts/xdg-copy.in	2015-09-18 10:18:14.648048300 -0500
@@ -96,7 +96,7 @@ case "$DE" in
     copy_kde "$source" "$dest"
     ;;
 
-    gnome*|cinnamon|lxde|mate|xfce)
+    gnome*|cinnamon|lxde|mate|xfce|cygwinx)
     copy_gnome "$source" "$dest"
     ;;
 
--- origsrc/xdg-utils/scripts/xdg-email.in	2015-09-18 04:14:56.766568900 -0500
+++ src/xdg-utils/scripts/xdg-email.in	2015-09-18 10:19:18.012094500 -0500
@@ -416,7 +416,7 @@ case "$DE" in
     open_gnome "${mailto}"
     ;;
 
-    gnome3|cinnamon|lxde|mate)
+    gnome3|cinnamon|lxde|mate|cygwinx)
     open_gnome3 "${mailto}"
     ;;
 
--- origsrc/xdg-utils/scripts/xdg-file-dialog.in	2015-09-18 04:14:28.000000000 -0500
+++ src/xdg-utils/scripts/xdg-file-dialog.in	2015-09-18 10:19:46.066657000 -0500
@@ -312,7 +312,7 @@ if [ "$action" = "openfilename" ]; then
         open_kde "$filename"
         ;;
 
-        gnome*|cinnamon|lxde|mate|xfce)
+        gnome*|cinnamon|lxde|mate|xfce|cygwinx)
         open_zenity "$filename"
         ;;
 
@@ -326,7 +326,7 @@ elif [ "$action" = "openfilenamelist" ];
         open_multi_kde "$filename"
         ;;
 
-        gnome*|cinnamon|lxde|mate|xfce)
+        gnome*|cinnamon|lxde|mate|xfce|cygwinx)
         open_multi_zenity "$filename"
         ;;
 
@@ -340,7 +340,7 @@ elif [ "$action" = "savefilename" ]; the
         save_kde "$filename"
         ;;
 
-        gnome*|cinnamon|lxde|mate|xfce)
+        gnome*|cinnamon|lxde|mate|xfce|cygwinx)
         save_zenity "$filename"
         ;;
 
@@ -354,7 +354,7 @@ elif [ "$action" = "directory" ]; then
         directory_kde "$filename"
         ;;
 
-        gnome*|cinnamon|lxde|mate|xfce)
+        gnome*|cinnamon|lxde|mate|xfce|cygwinx)
         directory_zenity "$filename"
         ;;
 
--- origsrc/xdg-utils/scripts/xdg-mime.in	2015-09-18 04:14:28.000000000 -0500
+++ src/xdg-utils/scripts/xdg-mime.in	2015-09-18 10:20:24.316514100 -0500
@@ -566,7 +566,7 @@ if [ "$action" = "info" ]; then
         info_kde "$filename"
         ;;
 
-        gnome*|cinnamon|lxde|mate|xfce)
+        gnome*|cinnamon|lxde|mate|xfce|cygwinx)
         info_gnome "$filename"
         ;;
 
--- origsrc/xdg-utils/scripts/xdg-open.in	2015-09-18 04:14:56.769069200 -0500
+++ src/xdg-utils/scripts/xdg-open.in	2015-09-18 10:21:06.785406900 -0500
@@ -412,7 +412,7 @@ case "$DE" in
     open_kde "$url"
     ;;
 
-    gnome*|cinnamon)
+    gnome*|cinnamon|cygwinx)
     open_gnome "$url"
     ;;
 
--- origsrc/xdg-utils/scripts/xdg-utils-common.in	2015-09-18 04:14:28.000000000 -0500
+++ src/xdg-utils/scripts/xdg-utils-common.in	2015-09-18 10:24:59.727986900 -0500
@@ -279,6 +279,9 @@ detectDE()
          XFCE)
            DE=xfce
            ;;
+         X-Cygwin)
+           DE=cygwinx
+           ;;
       esac
     fi
 
