--- origsrc/xdg-utils-1.1.1/scripts/xdg-utils-common.in	2016-03-09 22:31:45.701604200 -0600
+++ src/xdg-utils-1.1.1/scripts/xdg-utils-common.in	2016-03-09 22:34:37.459414700 -0600
@@ -318,7 +318,7 @@ detectDE()
       esac
     fi
 
-    if [ x"$DE" = x"" ]; then
+    if [ x"$DE$DISPLAY" = x"" ]; then
       # fallback to uname output for other platforms
       case "$(uname 2>/dev/null)" in 
         CYGWIN*)
