From abd6cef36874f73c5c1c4a734d83eba9d340afe4 Mon Sep 17 00:00:00 2001
From: Rex Dieter <rdieter@math.unl.edu>
Date: Sat, 19 Dec 2015 11:08:59 -0600
Subject: xdg-open: improve fallbacks, add open_generic (almost) everywhere (BR93442)


diff --git a/scripts/xdg-open.in b/scripts/xdg-open.in
index 92a5912..4578753 100644
--- a/scripts/xdg-open.in
+++ b/scripts/xdg-open.in
@@ -178,8 +178,10 @@ open_mate()
 {
     if gvfs-open --help 2>/dev/null 1>&2; then
         gvfs-open "$1"
-    else
+    elif mate-open --help >/dev/null 2>&1; then
         mate-open "$1"
+    else
+        open_generic "$1"
     fi
 
     if [ $? -eq 0 ]; then
@@ -191,7 +193,13 @@ open_mate()
 
 open_xfce()
 {
-    exo-open "$1"
+    if exo-open --help 2>/dev/null 1>&2; then
+        exo-open "$1"
+    elif gvfs-open --help 2>/dev/null 1>&2; then
+        gvfs-open "$1"
+    else
+        open_generic "$1"
+    fi
 
     if [ $? -eq 0 ]; then
         exit_success
@@ -202,7 +210,11 @@ open_xfce()
 
 open_enlightenment()
 {
-    enlightenment_open "$1"
+    if enlightenment_open --help 2>/dev/null 1>&2; then
+        enlightenment_open "$1"
+    else
+        open_generic "$1"
+    fi
 
     if [ $? -eq 0 ]; then
         exit_success
-- 
cgit v0.10.2

