--- origsrc/xdg-utils-v1.1.3/scripts/xdg-email.in	2020-12-19 21:58:50.958494200 +0900
+++ src/xdg-utils-v1.1.3/scripts/xdg-email.in	2020-12-19 22:01:32.129712700 +0900
@@ -455,7 +455,7 @@ fi
 if [ x"$BROWSER" = x"" ]; then
     BROWSER=www-browser:links2:elinks:links:lynx:w3m
     if has_display; then
-        BROWSER=x-www-browser:firefox:iceweasel:seamonkey:mozilla:epiphany:konqueror:chromium-browser:google-chrome:$BROWSER
+        BROWSER=epiphany:midori:konqueror:rekonq:qupzilla:$BROWSER
     fi
 fi
 
--- origsrc/xdg-utils-v1.1.3/scripts/xdg-open.in	2020-12-19 21:58:50.966494600 +0900
+++ src/xdg-utils-v1.1.3/scripts/xdg-open.in	2020-12-19 22:01:06.679257000 +0900
@@ -438,7 +438,7 @@ open_generic()
     if [ x"$BROWSER" = x"" ]; then
         BROWSER=www-browser:links2:elinks:links:lynx:w3m
         if has_display; then
-            BROWSER=x-www-browser:firefox:iceweasel:seamonkey:mozilla:epiphany:konqueror:chromium:chromium-browser:google-chrome:$BROWSER
+            BROWSER=epiphany:midori:konqueror:rekonq:qupzilla:$BROWSER
         fi
     fi
 
